app/assets/javascripts/discourse/app/controllers/users.js

```diff
@@ -19,6 +19,14 @@ export default Controller.extend({
 
   showTimeRead: equal("period", "all"),
 
+  @discourseComputed("group")
+  selectedGroupId(group) {
+    let selectedGroup = this.get("availableGroups").find(
+      (item) => item.name === group
+    );
+    return selectedGroup ? selectedGroup.id : -1;
```

Comment:
`-1` is abit magically here. What does the number represent?

Reply:
Nothing it was due to I missed something, changed it in the recent commit.